eva-accession-release-automation/publish_release_to_ftp/README_release_general_info.txt

```diff
@@ -138,6 +132,12 @@ deprecated due to the following reason(s):
 - The information was incomplete (Missing orientation, reference not among the
   alleles, RS ID variant type does not match type deducted from its SS IDs)
 
+- The position of the variant went beyond the chromosome length
+
+- The position of the variant was imprecise (formerly called multimapped)
```

Comment:
Should we add the phrase "multiple genomic locations" somewhere like dbSNP [does](https://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg19&g=snp150)?

Reply:
Done